exporter/exporterhelper/internal/retry_sender.go

```diff
@@ -111,7 +111,13 @@ func (rs *retrySender) Send(ctx context.Context, req request.Request) error {
 			backoffDelay = max(backoffDelay, throttleErr.delay)
 		}
 
-		if deadline, has := ctx.Deadline(); has && time.Until(deadline) < backoffDelay {
+		nextRetryTime := time.Now().Add(backoffDelay)
+		if !maxElapsedTime.IsZero() && maxElapsedTime.Before(nextRetryTime) {
```

Comment:
is this a separate bug? Is it covered by tests?

Reply:
https://github.com/open-telemetry/opentelemetry-collector/pull/11989/files#diff-8b81d46079780289189bba5dbaf864e9852872cbe614e031828a3870bb3f9950L78

Reply:
V5 does not have this logic supported.